### DISCUSSION TITLE
Wrong ID

### DISCUSSION THREAD

**Discussion by simonserranolara** [community/community#121108] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Through my company Accenture, we had access to a course where we were given the license to test Copilot. I received a license for the user with ID simon-serrano-lara_accent, but my ID is simonserranolara, so in the VSCode it fails to login.
What can I do?

**Comment by Akash1134** [comment, marked as answer]
Hi @simonserranolara ,

Please contact your administrator at Accenture who provided the license. They should be able to assist you in resolving the issue by providing access to the desired ID.

Thanks!

**Reply by simonserranolara** [reply]
I don't know who is the administrator. I received this invitation via email.
